src/index.js

```diff
@@ -0,0 +1,196 @@
+// install dependencies
+const inquirer = require("inquirer");
+const {
+  generateEngineerCard,
+  generateInternCard,
+  generateManagerCard,
+} = require("./util");
+const {
+  teamBuilderQuestions,
+  employeeTypeQuestion,
+  engineerQuestions,
+  internQuestions,
+  continueQuestion,
+  managerQuestion,
+} = require("./questions");
+const { writeToFile } = require("./file-gen");
+const { Engineer, Intern, Manager } = require("./lib/index");
+
+const generateHTMLfile = (teamName, teamMembers) => {
```

Comment:
This could probably be taken out into its own module to keep this file a bit cleaner.